elyra/pipeline/tests/test_validation.py

```diff
@@ -135,10 +135,9 @@ async def test_invalid_node_property_structure(monkeypatch, load_pipeline):
     pvm = PipelineValidationManager.instance()
```

Comment:
Does root_dir need to be conveyed when creating the validation manager instance?

Reply:
Ah, this is a hole in the coverage, tests were passing because I was monkeypatching past the filepath checks when using the manager and when when actually testing the filepath, I was calling the filepath test function directly and passing in the directory path. Adding another test case to fill this in.  Weird that the test failing here is passing locally.

Reply:
Yeah, I just added a comment that might help.  Since these are singletons, their behavior can be affected.